### PR TITLE
gave it an icon and a name

### PR DESCRIPTION
**Pull request by mLewisLogic**
@kfarst

### CODE REVIEW

File: alarm.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -553,6 +553,7 @@
 				IPHONEOS_DEPLOYMENT_TARGET = 8.1;
 				MTL_ENABLE_DEBUG_INFO = YES;
 				ONLY_ACTIVE_ARCH = YES;
+				PRODUCT_NAME = "";
```
Comment (kfarst) [inline alarm.xcodeproj/project.pbxproj:556]: @mLewisLogic no product name here?

File: alarm.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -588,6 +589,7 @@
 				GCC_WARN_UNUSED_VARIABLE = YES;
 				IPHONEOS_DEPLOYMENT_TARGET = 8.1;
 				MTL_ENABLE_DEBUG_INFO = NO;
+				PRODUCT_NAME = "";
```
Comment (kfarst) [inline alarm.xcodeproj/project.pbxproj:592]: @mLewisLogic same situation here?

Comment (mLewisLogic) [inline alarm.xcodeproj/project.pbxproj:592]: Weird. No idea. Guess it's worth setting.

Comment (mLewisLogic) [inline alarm.xcodeproj/project.pbxproj:592]: Fixed.

**Pull request merged**